src/actions/users.js

```diff
@@ -610,6 +610,21 @@ export function getMe() {
     };
 }
 
+export function updateServiceTermsStatus(serviceTermsId, accepted) {
+    return async (dispatch, getState) => {
+        await bindClientFunc(
+            Client4.updateServiceTermsStatus,
+            UserTypes.UPDATE_SERVICE_TERMS_STATUS_REQUEST,
+            UserTypes.UPDATE_SERVICE_TERMS_STATUS_SUCCESS,
+            UserTypes.UPDATE_SERVICE_TERMS_STATUS_FAILURE,
+            serviceTermsId,
+            accepted
+        )(dispatch, getState);
+        await getMe()(dispatch, getState);
```

Comment:
the same as above here, BUT, I'm not convinced you need to get the user profile yet again after logging in just to update 1 property, you could dispatch an action just to update that property in the current user, avoiding what is seems an unnecessary request to the server